### PR TITLE
Use https links

### PR DESCRIPTION
**Pull request by l2dy**
`find ./* -name '*.php' | xargs sed -i 's|href="http://|href="https://|g'`

### CODE REVIEW

**Comment by kecrily** [comment]
duplicate https://github.com/typecho/typecho/pull/1006 and seeing joyqi's reply.

**Comment by l2dy** [comment]
> duplicate #1006 and seeing joyqi's reply.

i18n files could have been updated with https://github.com/typecho/languages/pull/48. Any other reason it was not merged?

I could revert changes to links in `_e(...)` if i18n is still of concern.

**Comment by joyqi** [comment]
This pr will be merged in v1.2.1.

**Comment by l2dy** [comment]
Fixed merge conflict. @joyqi Could you merge this now?

**Comment by l2dy** [comment]
@joyqi Hi, could you merge this before merge conflicts happen again?

File: var/Typecho/Common.php
Comment (sy-records) [inline var/Typecho/Common.php:423]: 这的可以不用改

Comment (l2dy) [reply var/Typecho/Common.php]: Fixed.

File: var/Widget/Options/Discussion.php
Comment (sy-records) [inline var/Widget/Options/Discussion.php:130]:
```suggestion
            . _t('具体写法请参考 <a href="https://www.php.net/date">PHP 日期格式写法</a>.')
```

File: var/Widget/Options/Reading.php
Comment (sy-records) [inline var/Widget/Options/Reading.php:108]:
```suggestion
            . _t('请参考 <a href="https://www.php.net/date">PHP 日期格式写法</a>.')
```

File: usr/themes/default/footer.php
Comment (sy-records) [inline usr/themes/default/footer.php:9]:
```suggestion
    <?php _e('由 <a href="https://typecho.org">Typecho</a> 强力驱动'); ?>.
```

File: usr/themes/default/sidebar.php
Comment (sy-records) [inline usr/themes/default/sidebar.php:58]:
```suggestion
                <li><a href="https://typecho.org">Typecho</a></li>
```

@joyqi typecho网站的重定向有些问题，`https://www.typecho.org` 会重定向到 http://typecho.org

**Review by sy-records — CHANGES_REQUESTED** [review]

File: var/Widget/Options/Discussion.php
Comment (l2dy) [reply var/Widget/Options/Discussion.php:130]: What is this for?

Comment (sy-records) [reply var/Widget/Options/Discussion.php:130]: Let the php document choose its own language instead of forcing the Chinese document.

Comment (l2dy) [reply var/Widget/Options/Discussion.php:130]: I think it's out of scope for this PR, but I'll let the maintainers decide and have enabled "allow edits by maintainers".

File: usr/themes/default/sidebar.php
Comment (joyqi) [reply usr/themes/default/sidebar.php]: > @joyqi typecho网站的重定向有些问题，`https://www.typecho.org` 会重定向到 http://typecho.org

已修复

**Pull request merged**
